ios/NativeSigner/Screens/SettingsScreen.swift

```diff
@@ -69,7 +70,8 @@ struct SettingsScreen: View {
                                         hasPwd: false,
                                         identicon: [],
                                         seedName: "",
-                                        multiselect: false
+                                        multiselect: false,
+                                        secretExposed: false
```

Comment:
I see some code repetition here, I'll look into the way to removing that and making it more DRY, but not related to this PR 👍🏻